CRM/Contribute/BAO/Contribution.php

```diff
@@ -5245,4 +5248,44 @@ protected function assignTestValue($fieldName, &$fieldDef, $counter) {
     }
   }
 
+  /**
+   * Create Accounts Receivable financial trxn entry for Completed Contribution.
+   *
+   * @param array $params
```

Comment:
Are these the $params used to create/seach for a contribution? A bit of explanation would help.